js-arcade/tictactoe/TicTacToe.js

```diff
@@ -16,7 +19,55 @@ function TicTacToe() {
             this.xIsNext = !this.xIsNext;
         }
     };
-    this.isGameOver = function(x, y) {
+    this.isGameOver = function (x, y) {
+        let current_player = this.xIsNext ? true : false;
+        //Check row
+        for (i = 0; i < width; i++) {
+            if (!(this.squares[i][y] === current_player)) {
+                break
+            }
+            if (i === (width - 1)) {
```

Comment:
You don't actually need this check, since you break out of the for loop if you find a square that's not the current player's. It will only get to this line if it made it through the loop 3 times.